custom_components/smart_envi/config_flow.py

```
@@ -0,0 +1,676 @@
+        if self._device_id:
+            registry = entity_registry.async_get(self.hass)
+            for entry in entity_registry.async_entries_for_config_entry(registry, self.config_entry.entry_id):
+                if entry.domain == "climate":
+                    unique_id = entry.unique_id
+                    if unique_id.startswith(f"{DOMAIN}_"):
+                        device_id = unique_id.replace(f"{DOMAIN}_", "", 1)
+                        if device_id == self._device_id:
+                            entity_id = entry.entity_id
+                            state = self.hass.states.get(entity_id)
+                            if state:
+                                device_name = state.attributes.get("friendly_name", entity_id)
```

Ruff (0.14.8)

Local variable `device_name` is assigned to but never used

Remove assignment to unused variable `device_name`

(F841)